src/Controls/src/Core/PublicAPI/net-ios/PublicAPI.Shipped.txt

```diff
@@ -7447,7 +7447,6 @@ override Microsoft.Maui.Controls.Application.OnParentSet() -> void
 override Microsoft.Maui.Controls.Border.OnPropertyChanged(string? propertyName = null) -> void
 override Microsoft.Maui.Controls.BoxView.OnMeasure(double widthConstraint, double heightConstraint) -> Microsoft.Maui.SizeRequest
 override Microsoft.Maui.Controls.BoxView.OnPropertyChanged(string? propertyName = null) -> void
-override Microsoft.Maui.Controls.Button.ArrangeOverride(Microsoft.Maui.Graphics.Rect bounds) -> Microsoft.Maui.Graphics.Size
```

Comment:
For these you add a *REmoved* inside the Unshipped file. Check the Unshipped file for examples